relay-kafka/src/config.rs

```diff
@@ -124,6 +127,8 @@ pub struct TopicAssignments {
     /// COGS measurements.
     #[serde(alias = "shared-resources-usage")]
     pub cogs: TopicAssignment,
+    /// Feedback events topic name.
+    pub feedback: TopicAssignment,
```

Comment:
Probably missing a `#[serde(alias = "ingest-feedback-events")]`, assuming that's what the topic is called in the ops repo/Kafka schema.

Please double check this!